build-tools/packages/build-tools/src/fluidBuild/fluidPackageCheck.ts

```diff
@@ -488,22 +489,28 @@ export class FluidPackageCheck {
     private static checkLintScripts(pkg: Package, fix: boolean) {
         let fixed = false;
         if (pkg.getScript("build")) {
-            const hasPrettier = pkg.getScript("prettier");
-            const lintChildren = hasPrettier ? ["prettier", "eslint"] : ["eslint"];
-            if (this.checkChildrenScripts(pkg, "lint", lintChildren, false, fix)) {
-                fixed = true;
+            // const hasPrettier = pkg.getScript("prettier");
+            const hasLint = pkg.getScript("lint") === "npm run prettier && npm run eslint";
+
+            const lintChildren = hasLint ? ["prettier", "eslint"] : ["eslint"];
+
+            // npm run lint
+            // if hasLint has both "prettier && eslint"
+            if (hasLint) {
+                if (this.checkChildrenScripts(pkg, "lint", lintChildren, false, fix)) {
```

Comment:
This check is the same in both conditional branches, so I think we can remove the outer if-else (unless there is supposed to be a difference in the checks).

Reply:
The else branch should have been deleted 😅. Thank you!